extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/SpringDataJPAProcessor.java

```diff
@@ -66,6 +80,40 @@ void build(CombinedIndexBuildItem index,
                 interfacesExtendingCrudRepository, indexIndex);
     }
 
+    private void detectAndLogSpecificSpringPropertiesIfExist() {
+        Config config = ConfigProvider.getConfig();
+        Map<String, String> springJpaToQuarkusOrmPropertiesMap = new HashMap<>();
+        springJpaToQuarkusOrmPropertiesMap.put("spring.jpa.show-sql", "quarkus.hibernate-orm.log.sql");
+        springJpaToQuarkusOrmPropertiesMap.put("spring.jpa.properties.hibernate.dialect", "quarkus.hibernate-orm.dialect");
+        springJpaToQuarkusOrmPropertiesMap.put("spring.jpa.properties.hibernate.dialect.storage_engine",
+                "quarkus.hibernate-orm.dialect.storage-engine");
+        springJpaToQuarkusOrmPropertiesMap.put("spring.jpa.generate-ddl", "quarkus.hibernate-orm.database.generation");
+
+        Iterable<String> iterablePropertyNames = config.getPropertyNames();
+        List<String> propertyNames = new ArrayList<String>();
+        iterablePropertyNames.forEach(propertyNames::add);
+        List<String> springProperties = propertyNames.stream().filter(s -> pattern.matcher(s).matches()).collect(toList());
+
+        if (!springProperties.isEmpty()) {
+            List<String> springWithQuarkusEquivalences = springProperties.stream()
+                    .filter(s -> springJpaToQuarkusOrmPropertiesMap.containsKey(s)).collect(Collectors.toList());
+
+            String notSupportedProperties = springWithQuarkusEquivalences.stream()
+                    .map(d -> "\t- " + d + " should be replaced by " + springJpaToQuarkusOrmPropertiesMap.get(d))
+                    .collect(Collectors.joining("\n"));
+
+            List<String> others = springProperties.stream().filter(s -> !springJpaToQuarkusOrmPropertiesMap.containsKey(s))
+                    .collect(Collectors.toList());
+
+            notSupportedProperties = notSupportedProperties + "\n"
+                    + others.stream().map(d -> "\t- " + d).collect(Collectors.joining("\n"));
+
+            LOGGER.warnf(
+                    "Quarkus does not support the following Spring Boot configuration properties :%n%s",
```

Comment:
```suggestion
                    "Quarkus does not support the following Spring Boot configuration properties:%n%s",
```
No space before colons in English :).

Reply:
A space after would be nice!

Reply:
Done!